install/docker/index.markdown

```diff
@@ -29,54 +29,47 @@ If you are running a recent version of Ubuntu (e.g. 14.04, 16.04) it can be as s
 
 And you will likely need to log out and back into your user account for the changes to take affect.
 
-## Pulling MoveIt! Images
+## Running MoveIt! Containers
 
-Now that you have Docker installed, pull down a MoveIt! container image:
+### New Users
 
-    docker pull moveit/moveit
+To run a Debian-installed container of MoveIt! will graphics support:
 
-This will pull the latest tagged image of MoveIt! from Docker Hub onto your local host machine.
+    wget https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/.docker/gui-docker gui-docker
+    . ./gui-docker -ti --rm moveit/moveit:kinetic-release /bin/bash
```

Comment:
Sourcing this is a little weird. I think it would be better to make it executable. 

```wget https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/.docker/gui-docker gui-docker && chmod +x gui-docker```

Then executing is just
`./gui-docker -ti --rm moveit/moveit:kinetic-release /bin/bash`

Reply:
done